decorate_test.go

```diff
@@ -185,6 +186,55 @@ func TestDecorateSuccess(t *testing.T) {
 		defer app.RequireStart().RequireStop()
 	})
 
+	t.Run("decorator with soft value group", func(t *testing.T) {
+		app := fxtest.New(t,
+			fx.Provide(
+				fx.Annotate(
+					func() (string, int) { return "cheeseburger", 15 },
+					fx.ResultTags(`group:"burger"`, `group:"potato"`),
+				),
+			),
+			fx.Provide(
+				fx.Annotate(
+					func() (string, int) { return "mushroomburger", 35 },
```

Comment:
non-review comment: mushroom burger...? 🤣

Reply:
While I was writing this I couldn't remember the name of another burger hahaha